README.adoc

```diff
@@ -231,9 +280,13 @@ include::finish/src/test/java/it/io/openliberty/guides/health/HealthTest.java[ta
 
 Let's break down the test cases:
 
-- The [hotspot=27-30 file=0]`testIfServicesAreUp()` test case compares the generated health report 
-with the actual status of the services.
-- The [hotspot=33-40 file=0]`testIfInventoryServiceIsDown()` test case puts the `inventory` service 
+- The [hotspot=25-33 file=0]`testIfServicesAreUp()` test case compares the generated health report 
+with the actual status of the services. 
+- The [hotspot=36-42 file=0]`testReadiness()` test case compares the generated health report for the 
+readiness checks with the actual status of the services. 
+- The [hotspot=45-51 file=0]`testLiveness()` test case compares the generated health report for the 
+liveness checks with the actual status of the services. 
+- The [hotspot=54-68 file=0]`testIfInventoryServiceIsDown()` test case puts the `inventory` service 
 in maintenance by setting the [hotspot=3 file=1]`io_openliberty_guides_inventory_inMaintenance` 
 property to `true` and then compares the generated health report with the actual status of 
 the service.
```

Comment:
Change
"The [hotspot=54-68 file=0]`testIfInventoryServiceIsDown()` test case puts the `inventory` service in maintenance by setting the [hotspot=3 file=1]`io_openliberty_guides_inventory_inMaintenance` property to `true` and then compares the generated health report with the actual status of the service."
To
"The [hotspot=54-68 file=0]`testIfInventoryServiceIsDown()` test case puts the `inventory` service in maintenance by setting the [hotspot=3 file=1]`io_openliberty_guides_inventory_inMaintenance` 
property to `true` and comparing the generated health report with the actual status of the services."